src/main/java/io/confluent/connect/jdbc/source/JdbcSourceTask.java

```diff
@@ -151,6 +151,12 @@ public void start(Map<String, String> properties) {
       boolean mapNumerics
           = config.getBoolean(JdbcSourceTaskConfig.NUMERIC_PRECISION_MAPPING_CONFIG);
 
+      TimestampHelper timestampHelper = new MultiColumnTimestampHelper(timestampColumn);
```

Comment:
Should we just set to `null` initially followed by an if-else? Maybe the instantiation here is not worth it, given we might discard it below.